vassal-app/src/main/java/VASSAL/tools/image/svg/SVGRenderer.java

```diff
@@ -162,7 +162,7 @@ public BufferedImage render(double angle, double scale, Rectangle2D aoi) {
     px = new AffineTransform(px);
     px.scale(scale, scale);
 
-    final Rectangle2D rect = new Rectangle2D.Float(0, 0, defaultW, defaultH);
+    new Rectangle2D.Float(0, 0, defaultW, defaultH);
```

Comment:
This line is cruft which should have been removed when the line which used `rect` was removed. It can be deleted.